packages/react/src/components/DotcomShell/README.md

```diff
@@ -0,0 +1,43 @@
+# DotcomShell
+
+> The DotcomShell component includes the `Masthead` and `Footer` components, all
+> wrapped in a UI shell using Carbon's grid.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { DotcomShell } from '@carbon/ibmdotcom-react';
+import '@carbon/ibmdotcom-styles/scss/components/masthead/masthead.scss';
+import '@carbon/ibmdotcom-styles/scss/components/footer/footer.scss';
```

Comment:
Does it make sense to create a `_dotcomshell.scss` under `@carbon/ibmdotcom-styles`, so we can include all of this for them?

Reply:
@photodow Yes definitely. I already have `DotcomShell` in the styles package. I'll consolidate masthead/footer into that.